### ISSUE TITLE
Small screen size optimization

### ISSUE THREAD

**Issue by ellyasafaa** [onaio/rdt-standard#811]
On Samsung J5 Pro, the screen will get cut. Could we optimize the screen size for 5-inch phones?

![Screenshot_20210219-132049_RDT Open Reader](https://user-images.githubusercontent.com/73918009/108791590-039a3b80-75b2-11eb-8c74-880a6090fac1.jpg)

**Comment by vincent-karuri** [comment]
Are you able to scroll? Is this output from the J5? If not, please post a picture of what exactly happens.

**Comment by ellyasafaa** [comment]
@vincent-karuri Yes, I am able to scroll and this is the output from the J5. However, could we optimize it and adjust the screen size on that device? If the user is not aware that the page can be scrolled, they won't scroll anyway.

**Comment by vincent-karuri** [comment]
This was designed to be scrollable since it's possible to add other forms.

**Comment by mwkhan1983** [comment]
@ellyasafaa this is out of scope. Please assign some one from your team to work on this. Thanks
